MetaMask/metamask-mobile
# fix: upgrade permission-controller to 8.0.0

## **Description**

Upgrades the `permission-controller` from 7.1.0 --> 8.0.0

## **Related issues**

Fixes: [1695](https://github.com/MetaMask/mobile-planning/issues/1695)

## **Manual testing steps**

* Connect to app
    * Single account
    * Multiple accounts
* With multiple connected accounts
    * Disconnect Account
        * Next connected account should be selected
* Open a new tab
    * Connect to test app
    * https://metamask.github.io/test-dapp
        * Connect accounts
        * Disconnect account

## **Recordings**
### Connect Dapp Single Account

https://github.com/MetaMask/metamask-mobile/assets/141057783/9ca0afd9-5b61-4f2d-ba12-e7b1bd12e3b4


### Connect Dapp Multiple Accounts

https://github.com/MetaMask/metamask-mobile/assets/141057783/49b1d67c-6b2e-4203-b172-53420a8753d4


### TestDapp Connect Account

https://github.com/MetaMask/metamask-mobile/assets/141057783/fa896121-34b7-4c2f-8680-6e85640e60cf


### TestDapp Connect/Disconnnect/GET Account

https://github.com/MetaMask/metamask-mobile/assets/141057783/ded58259-4679-4790-85f7-0a34a3b51ca3



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
